Dockerfile

```diff
@@ -56,6 +56,7 @@ ENV APP_BUILD_DATE=${BUILD_DATE}
 ENV APP_BUILD_TAG=${BUILD_TAG}
 
 RUN python3 manage.py migrate --no-input \
+    && python3 manage.py cleardata \
```

Comment:
Can you add a comment in the description on why we have to add this line?
Just for future reference

Reply:
@ddippolito that comments added